.gitlab/macrobenchmarks.yml

```diff
@@ -1,7 +1,7 @@
 variables:
-  BENCHMARKS_CI_IMAGE: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/benchmarking-platform:go-go-prof-app
+  BENCHMARKS_CI_IMAGE: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/benchmarking-platform:go-go-prof-app-and-serviceextensions-0001
```

Comment:
@e-n-0 How's this image maintained?

Reply:
This image is built from the branch [go/go-prof-app](https://github.com/DataDog/benchmarking-platform/tree/go/go-prof-app) of the benchmarking-platform repo.
The image is only built one time (by running the `ci-images` CI stage).
The image can be rebuild anytime we want, if for example we want to have a newer version of Envoy.